testutils/infra/container_manager/base.py

```diff
@@ -12,3 +15,16 @@ def teardown(self):
         """Stops the running containers
         """
         pass
+
+    def execute(self, container_id, cmd):
+        """Executes the given cmd on an specific container
+        """
+        pass
```

Comment:
Minor note, I would replace `pass` with `raise NotImplementedError` in order to catch, well, not-implemented-errors in classes providing the actual implementation based on this abstract class.

Reply:
Good idea. I will do it in all methods.